packages/pluggableWidgets/slider-native/src/Slider.tsx

```diff
@@ -21,8 +22,10 @@ export function Slider(props: Props): ReactElement {
     const validProps = validationMessages.length === 0;
     const editable = props.editable !== "never" && !props.valueAttribute.readOnly && validProps;
     const styles = flattenStyles(defaultSliderStyle, props.style);
+    // We have to fix the decimal count ourselves because of an unresolved bug in the library: https://github.com/ptomasroos/react-native-multi-slider/issues/211
+    const decimalCount = useCallback((value: Big): number => value?.toString().split(".")?.[1]?.length || 0, []);
```

Comment:
Since you have `value?.toString()...` it seems that the function's argument type should be `Option<Big>` or am I missing something?

Reply:
Yes I agree, added